VSDiagnostics/VSDiagnostics/VSDiagnostics/Diagnostics/General/ImplementEqualsAndGetHashCode/ImplementEqualsAndGetHashCodeAnalyzer.cs

```diff
@@ -67,72 +71,27 @@ private void GetEqualsAndGetHashCodeSymbols(SemanticModel model, out IMethodSymb
             }
         }
 
-        private bool TypeMembersContainOverridenEqualsOrGetHashCode(SemanticModel model, SyntaxList<SyntaxNode> members, IMethodSymbol objectEquals, IMethodSymbol objectGetHashCode)
+        private bool MembersContainNonStaticFieldOrProperty(ImmutableArray<ISymbol> members)
```

Comment:
Because it suddenly became much simpler to check the stuff with the properties in `ISymbol` and `IPropertySymbol`.